make/stackrox.mk

```diff
@@ -37,7 +37,7 @@ posttest:
 .PHONY: test-common
 test-common:
 	@echo "+ $@"
-	$(SILENT)$(TOPLEVEL)/scripts/go-test.sh -cover $(TESTFLAGS) -v $(shell go list -e ./... | grep -v generated | grep -v vendor) 2>&1 | tee test.log
+	@GOTAGS=$(GOTAGS),test,test_e2e $(SILENT)$(TOPLEVEL)/scripts/go-test.sh -cover $(TESTFLAGS) -v $(shell go list -e ./... | grep -v generated | grep -v vendor) 2>&1 | tee test.log
```

Comment:
```suggestion
	$(SILENT)GOTAGS=$(GOTAGS),test,test_e2e $(TOPLEVEL)/scripts/go-test.sh -cover $(TESTFLAGS) -v $(shell go list -e ./... | grep -v generated | grep -v vendor) 2>&1 | tee test.log
```

Reply:
I find it a bit strange that every other line starts with @ but the one with $(SILENT) does not, but it's the same in the test_integration targets so it should be correct. I'm not familiar enough with Makefile syntax to know why though, I'll read up on that later.

Anyways, making this change

Reply:
`@` stops the command from being printed. The `SILENT` variable contains the `@` by default. If you were to modify the `SILENT` variable to ` `, the Makefile would be more verbose. I think this is for debugging purposes.